### PR TITLE
Update Docker PaaS tutorials to include all new changes in the gamma 

### PR DESCRIPTION
**Pull request by brouberol**
:warning: This PR needs to be merged when the Docker PaaS gamma is announced.

When the PR is considered mergeable, please wait for my go :)

### CODE REVIEW

File: en/_posts/2016-04-20-introduction-to-docker-with-mesos-marathon.md [hunk 1]
```diff
@@ -24,25 +24,24 @@ If you want more information about the Apache Mesos architecture, please read th
...
+[Marathon](https://mesosphere.github.io/marathon/) is an Apache Mesos framework to run long-running processes, such as Docker containers. It provides a lot of features like framework high availibility through an active/passive system, anti-affinity constraints to ensure containers are launched on different hosts or racks, application health checks, etc. You can manage your applications and containers using a web UI or an API, allowing you to easily launch new containers, scale your applications, and more.
 
 #What do we provide?
 
-The Docker PaaS lab is based on all these software stacks, adding some other cool stuff, like redirecting your application Logs to our PaaS Logs, and your application metrics our PaaS Timeseries.. Our lab is stack managed, it means that **we** deploy and manage your infrastructure.
+The Docker PaaS lab is based on all these software stacks, adding some other cool stuff, like redirecting your application Logs to our [PaaS Logs](https://www.runabove.com/paas-logs.xml), and your application metrics the [DBaaS Timeseries](https://www.ovh.com/fr/dbaas/timeseries/). Our lab is stack managed, it means that **we** deploy and manage your infrastructure.
```
Comment (yadutaf) [inline en/_posts/2016-04-20-introduction-to-docker-with-mesos-marathon.md]: "Timeseries" is spelled "Time series" below

Comment (brouberol) [inline en/_posts/2016-04-20-introduction-to-docker-with-mesos-marathon.md]: Good spot! Updating/

**Comment by brouberol** [comment]
I rollbacked some modifications on the load-balancer article (we're not yet prodding the new LB) . When travis approves the build, you have my go :)

**Comment by yadutaf** [comment]
Go go go !

**Pull request merged**